server/src/common-types/gql.ts

```diff
@@ -1,13 +1,15 @@
 import { Request as ExpressRequest, Response } from 'express';
 
-import type { User } from '../controllers/Auth/middleware';
+import type { User, Events } from '../controllers/Auth/middleware';
 
 export interface GQLCtx {
```

Comment:
Can we have a different name than `GQLCtx` 🥺

Reply:
ResolverCtx?

Reply:
Sure, 👍 it took me a minute to understand it's related to graphql. 

I like the new name more💚